src/System.Private.CoreLib/shared/System/Buffers/Text/FormattingHelpers.CountDigits.cs

```diff
@@ -128,5 +128,34 @@ public static int CountHexDigits(ulong value)
 
             return digits;
         }
+
+        
+        // Counts the number of trailing '0' digits in a decimal number.
+        // e.g., value =      0 => retVal = 0, valueWithoutTrailingZeros = 0
+        //       value =   1234 => retVal = 0, valueWithoutTrailingZeros = 1234
+        //       value = 320900 => retVal = 2, valueWithoutTrailingZeros = 3209
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static int CountDecimalTrailingZeros(uint value, out uint valueWithoutTrailingZeros)
```

Comment:
Purely out of curiosity, is there any reason to have a unit test for this function? You have the documentation above, but since it is public is a unit test warranted?

Reply:
I don't think it's necessary.  This method also isn't new: it's just moved here from corefx.  Once this mirrors over to corefx, I'll consolidate, removing the separate copy that's over there.